Tools/AP_Periph/AP_Periph.cpp

```diff
@@ -100,6 +100,10 @@ void AP_Periph_FW::init()
 
     can_start();
 
+#if AP_NETWORKING_ENABLED
```

Comment:
Sadly, I believe we're requiring `#ifdef HAL_PERIPH_ENABLE_NETWORKING`.  You will need to turn `AP_NETWORKING_ENABLED` on in `chibios_hwdef.py` if that thing is defined.  Don't blame me, I'm not keen on the rule either.